subsys/bluetooth/controller/Kconfig

```diff
@@ -112,6 +112,23 @@ config BLUETOOTH_CONTROLLER_LE_PING
 	help
 	  Enable support for Bluetooth v4.1 LE Ping feature in the Controller.
 
+config BLUETOOTH_CONTROLLER_PRIVACY
```

Comment:
Add entries for new Kconfigs in a tests/bluetooth/init/prj_controller_*.conf as appropriate.